k8s/deployments/core-secondary-deployment.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: apps/v1
```

Comment:
Can (but should we ?) we inherit from core-deployment.yaml and override the variables we need ?

Reply:
I wish but sadly I don't think you can do that (need specialized tools that we don't use to "build" yaml -- for now we're just duplicating when needed)